pkg/autoscaler/scaling/autoscaler.go

```diff
@@ -103,7 +103,13 @@ func newAutoscaler(
 	} else {
 		pkgmetrics.Record(reporterCtx, panicM.M(0))
 	}
-
+	maxPanicPods := int32(curC)
```

Comment:
That's not correct. Since if I am scaled to, say 1, and initialScale is 10 and autoscaler restarts — this will scale to 10.

Reply:
Yes that's a good point. I updated how the InitialScale field is set on DeciderSpec. Maybe we should rename the field InitialScale to something like AutoscalerInitialScale?

Reply:
I think you don't need to thread it to the autoscaler in general. I think (just like minScale) this should be an artifact of KPA.

Reply:
Hmm I'm not sure if I understand. The autoscaler currently doesn't have access to KPA. Is there a way to start in panic mode outside of autoscaler?